app/models/course/assessment/question/text_response_comprehension_solution.rb

```diff
@@ -32,4 +34,13 @@ def strip_whitespace_solution
   def strip_whitespace_solution_lemma
     solution_lemma.each(&:strip!)
   end
+
+  def validate_solution_lemma_not_empty
+    errors.add(:solution, :solution_lemma_empty) if solution_lemma.empty?
```

Comment:
Is it better to add something like `f.error :solution_lemma` to `_comprehension_solution_fields.html.slim` to surface the error?

Reply:
Agreed, since the error is on `solution_lemma` being blank, and not on `solution`.